static/.htaccess

```diff
@@ -1,3 +1,31 @@
+# Welcome to the redirect party! 
+#  ____  ____  ____  __  ____  ____  ___  ____    ____   __   ____  ____  _  _  _   
+# (  _ \(  __)(    \(  )(  _ \(  __)/ __)(_  _)  (  _ \ / _\ (  _ \(_  _)( \/ )/ \  
+#  )   / ) _)  ) D ( )(  )   / ) _)( (__   )(     ) __//    \ )   /  )(   )  / \_/  
+# (__\_)(____)(____/(__)(__\_)(____)\___) (__)   (__)  \_/\_/(__\_) (__) (__/  (_)  
+#
```

Comment:
oof, this looks bad in the diffed version. Standalone it is a lot more fun. I can remove it if you want.

Reply:
Keep it! We should have a little fun every now and then.